### ISSUE TITLE
!mp invite command invites to wrong lobby if more than one were created in same second.

### ISSUE THREAD

**Issue by Master-chan** [ppy/osu-stable-issues#757]
Title. Its pretty old bug, I reported it on forums last year and decided to move it here: https://osu.ppy.sh/community/forums/topics/1055557?start=7426128

Latest example:
https://osu.ppy.sh/community/matches/83554692
https://osu.ppy.sh/community/matches/83554693

Players were invited to first lobby:
![изображение](https://user-images.githubusercontent.com/1059861/118882860-0cc74a00-b90e-11eb-9718-584c7a19a532.png)

**Comment by smoogipoo** [comment]
I've fixed this in bancho for the next deploy.

**Comment by Master-chan** [comment]
I don't know if you deployed the fixed version or not but this is still happening:

https://osu.ppy.sh/community/matches/84641878
https://osu.ppy.sh/community/matches/84641880

**Comment by peppy** [comment]
No, it has not been deployed. Please wait, and check @osustatus on twitter for any signs of deploys.
